src/amo/components/App/index.js

```diff
@@ -137,8 +137,8 @@ export class AppBase extends React.Component<Props> {
         `Auth token "${encodedTimestamp}" triggered this base62 error: "${base62Error}"`);
       return;
     }
-    // If the encoded timestamp was malformed it will be 0 or negative.
-    if (createdAt <= 0) {
+    // If the encoded timestamp was malformed it will be 0 or negative or NaN
+    if (createdAt <= 0 || Number.isNaN(createdAt)) {
```

Comment:
Should these conditions be in the reverse order to that `Number.isNaN(createdAt)` shortcuts before checking `createdAt <= 0`, or does that not really matter?

Reply:
It can still return 0 - I added a test to cover that case.